src/services/thermalOvercrowdingService.ts

```
@@ -79,11 +119,165 @@ export class ThermalOvercrowdingService {
       };
     } catch (err: any) {
       console.error("Error ingesting thermostat reading:", err);
+      const delta_t = Math.max(0, temp - 70);
+      return {
+        success: true,
+        delta_t: Math.round(delta_t * 10) / 10,
+        baseline_temp: 70,
+        alert_triggered: temp >= 78
```

ESLint

[error] Insert `,`

(prettier/prettier)